pkg/kubernetes/selinux.go

```diff
@@ -0,0 +1,82 @@
+package kubernetes
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/suse-edge/edge-image-builder/pkg/http"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+	"go.uber.org/zap"
+)
+
+const (
+	rke2SELinuxPolicyReleaseURL = "https://github.com/rancher/rke2-selinux/releases/download/%s/%s"
+	k3sSELinuxPolicyReleaseURL  = "https://github.com/k3s-io/k3s-selinux/releases/download/%s/%s"
```

Comment:
@rdoxenham @hardys Apologies if this is a question that's been answered already, but is pulling this from github instead of something like OBS going to be a problem for us from a release artifacts point of view?

Reply:
These RPMs don't exist in OBS and we don't really want to maintain a fork.

Reply:
I discussed this with @atanasdinov - we should consume RPM releases from rke2/k3s instead of the GH releases, these will be signed artefacts with a discoverable version, and there will presumably be a day-2 strategy to upgrade them alongside the other rke2/k3s components which we don't have to own directly.